site/app/templates/admin/users/GraderListForm.twig

```diff
@@ -3,8 +3,9 @@
 {% block title %}Upload Grader List{% endblock %}
 {% block body %}
     <p>
-        Format your grader data as an .xlsx or .csv file with {{ use_database ? 8 : 7 }} columns:<br>
+        Format your grader data as an .xlsx or .csv file with {{ use_database ? 8 : 7 }} columns :<br>
         &emsp;username, first name, last name, email, grader group, {{ use_database ? "password, " : "" }}preferred first name, preferred last name<br>
```

Comment:
While we're here, can you modify this to be a numbered list like the student upload form?

Reply:
Done